src/api/classinfo.js

```diff
@@ -0,0 +1,31 @@
+import { instance } from '.';
+
+export const getClassinfo = async () => {
+  try {
+    const response = await instance.get(`/class-info`);
+    return response.data;
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export const updateClassinfo = async classInfo => {
+  const accessToken = localStorage.getItem('token');
+
+  if (!accessToken) {
+    console.error('No access token found, please log in.');
+    return;
+  }
+
+  try {
+    const response = await instance.put(`/class-info`, classInfo, {
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+      },
+    });
```

Comment:
```suggestion
    const response = await authInstance.put(`/class-info`, classInfo);
```
헤더에 토큰 들어가야 하는 건 그냥 instance말고 authInstance 사용하면 거기에 포함되어 있어요